frontend/src/styles/utils/decorations.scss

```
@@ -33,6 +33,50 @@ body {
   &-hidden { overflow: hidden; }
 }
 
+/////////////////////////
+//   BORDER CLASSES    //
+/////////////////////////
+$spacing-xxs:  0.1px;
+$spacing-xs:  0.5px;
+$spacing-s:   1px;
+$spacing-m:   2px;
+$spacing-l:   4px;
+$spacing-xl:  8px;
+
+// border-radius
+.border{
+  &-m{
+    border-radius: $spacing-m $spacing-m $spacing-m $spacing-m;
+  }
+  &-l{
+    border-radius: $spacing-l $spacing-l $spacing-l $spacing-l;
+  }
+}
+
+// Message / Highlight styling
+.highlight{
+  &-blue{
```

space in between class name and brace 😋 
```suggestion
  &-blue {
```